.gitignore

```diff
@@ -5,4 +5,4 @@
 dist
 docs/_build
 build/
-venv/
+/venv*/
```

Comment:
Is the leading slash correct here? Should it just be `venv*/` This isn't a regular expression, it's a glob (I thought)

Reply:
The leading slash in `.gitignore` implies that it should only match `venv*/` directories that directly neighbor the `.gitignore` file.

e.g. it will ignore `pycodestyle/venv-biscuit/`, but not `pycodestyle/docs/venv-cookie`.

Reply:
Citing da sources:
https://git-scm.com/docs/gitignore#_pattern_format

> A leading slash matches the beginning of the pathname. For example, "/*.c" matches "cat-file.c" but not "mozilla-sha1/sha1.c".